kitchensink/src/app/static/foundations/ColorsPage.jsx

```diff
@@ -92,7 +92,7 @@ const ColorsPage = () => (
           description=""
           rowHeader={{
             colors: [{
-              colorName: 'Information', name: 'blue-500', darkFont: false, bordered: false,
+              colorName: 'Information', name: 'blue-500', darkFont: false, bordered: false, infoText: 'blue-500',
```

Comment:
Why? what is the difference to name?

Reply:
name is a variable and the colorbox cannot be displayed without it. infoText contains text that describes the color and it can be omitted.